services/Netatmo.js

```diff
@@ -0,0 +1,70 @@
+const util = require('util');
+const qs = require('qs');
+const axios = require('axios');
+const BASE_URL = 'https://api.netatmo.net';
+
+class Netatmo {
+
+	/**
+	 * @constructor
+	 */
+	constructor() {
+	}
+
+	/**
+	 * Add webhook
+	 * @param args
+	 * @param webhook
+	 */
+	addWebhook(args, webhook) {
+		const data = {
+			client_id: args.client_id,
+			client_secret: args.client_secret,
+			username: args.username,
+			password: args.password,
+			scope: args.scope,
+			grant_type: args.grant_type,
+		};
+
+		var url = util.format('%s/oauth2/token', BASE_URL);
```

Comment:
Either use `let` or `const`. `var` is a legacy statement valid for both local and global variables, which can be confusing.